bloom-housing/bloom
# 3218/updated design tokens

## Issue Overview

This PR addresses #3218 and #3175


## Description

These are the necessary Bloom core updates to use design token changes in bloom-housing/ui-components#6

## How Can This Be Tested/Reviewed?

You would need to check out the PR in `ui-components` locally, then link that page to use in the Next.js apps.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
